cmd/mattermost/commands/init.go

```diff
@@ -45,6 +45,7 @@ func InitDBCommandContext(configDSN string) (*app.App, error) {
 
 	if model.BuildEnterpriseReady == "true" {
 		a.Srv().LoadLicense()
+		a.InitEnterprise()
```

Comment:
Why not just execute here `a.InitServer()`, actually, I think this should be call outside the `BuildEnterpriseReady` if block.